### ISSUE TITLE
Microsoft.Windows.Compatibility pulls in unlisted non-Microsoft MEF1 package

### ISSUE THREAD

**Issue by nblumhardt** [dotnet/corefx#27498]
Guessing this may have already been noted elsewhere, but took me by surprise - the MEF1 work for the Windows Compatibility Pack hasn't been done yet (mentioned on #24909), but _Microsoft.Windows.Compatibility_ still declares it as a package dependency.

The NuGet client falls back to installing this unlisted non-Microsoft package uploaded in 2010:

https://www.nuget.org/packages/System.ComponentModel.Composition

**Comment by nblumhardt** [comment]
I'm used to this being a warning, but seems at least in my build it's an error, so the unlisted package doesn't end up being installed. Not sure if this will be the case for everyone, but updated the issue title to reflect this.

**Comment by nblumhardt** [comment]
Sorry, another addition - I didn't realize the package has been rev'd; the one I'm installing is:

Microsoft.Windows.Compatibility 2.0.0-preview1-25914-04

**Comment by nblumhardt** [comment]
Sorry, another addition - I didn't realize the package has been rev'd; the one I'm installing is:

Microsoft.Windows.Compatibility 2.0.0-preview1-26216-02

**Comment by danmosemsft** [comment]
@weshaggard

**Comment by superriva** [comment]
@nblumhardt 
https://dotnet.myget.org/feed/dotnet-core/package/nuget/System.ComponentModel.Composition

**Comment by weshaggard** [comment]
@nblumhardt Yes we are missing System.ComponentModel.Composition on nuget.org right now as someone was sitting on that package id and we are trying to get ownership of it. Until that happens you can fix this by adding our myget feed to your package sources.

**Comment by nblumhardt** [comment]
Thanks @weshaggard @superriva - I don't actually need the package, just wanted to make sure no one inadvertently installed "mystery" code via this route :+1:

**Comment by NBaron** [comment]
@superriva I am trying to use the package on myget in a NetStandard2 library.
I am porting from a NetFramework4.6.1 library using MEF1.
Everything looks good till I get to the use of RegistrationBuilder which is a CustomReflectionContext (System.ComponentModel.Composition.Registration).
It is apparently not present in the myget package and I would like to know how can we achieve something similar; like filtering the exports.

E.g.:
```
var conventions = new RegistrationBuilder();
conventions.ForTypesDerivedFrom<ILogger>()
                    .ExportInterfaces(t => t != typeof(ILogger)); // Exporte le logger
conventions.ForTypesMatching(t => t.IsClass && !t.IsAbstract && PLAN_TYPE.IsAssignableFrom(t))
                   .ImportProperties(pi => PLAN_VALUE_TYPE.IsAssignableFrom(pi.PropertyType));

var globalCatalog = new AggregateCatalog(
                   globalAssemblies.Select(a => new AssemblyCatalog(a,  conventions)));
```

Thanks in advance.
